### DISCUSSION TITLE
Create Example Calling AWS SDK in Lambda

### DISCUSSION THREAD

**Discussion by CodingFlow** [leptos-rs/leptos#2306] [Q&A]
I'm currently trying to use the AWS Rust SDK to call AWS services from my SSR Leptos app deployed on Lambda, but running into build time issues (Looks like [this one](https://github.com/seanmonstar/reqwest/issues/1300) that's closed 🤔). It would be great to have an example app that is working for this, perhaps an extension to the [aws-starter](https://github.com/leptos-rs/start-aws).

**Comment by gbj** [comment]
If it actually looks like the closed issue with errors like this
```
12 | use crate::sys::IoSourceState;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^ no `IoSourceState` in `sys`
```
or errors referring to `mio`, then it is almost certainly the result of trying to add a dependency that can't be compiled to WebAssembly, probably something in the Tokio universe. [See here in the docs](https://book.leptos.dev/ssr/24_hydration_bugs.html?highlight=mio#not-all-server-code-can-run-on-the-client).
